.pipelines/build-pipelines.yml

```diff
@@ -38,15 +38,25 @@ steps:
 # We cannot set this in variables section above because $(isNugetRelease) 
 # is not available at pipeline compilation time.
 - bash: |
-    echo ISNUGETRELEASE = $ISNUGETRELEASE
+    echo IsNugetRelease = $ISNUGETRELEASE
+    echo IsReleaseCandidate = $ISRELEASECANDIDATE
```

Comment:
will these be 2 new variables to the pipeline - ISRELEASECANDIDATE
RELEASE_CANDIDATE_NUMBER?

Reply:
Yes

Reply:
update to add only 1 new variable.